sdk/repo.go

```diff
@@ -153,5 +157,10 @@ func RepoSync(chroot string, force bool) error {
 	if force {
 		args = append(args, "--force-sync")
 	}
-	return enterChroot(chroot, chrootRepoRoot, args...)
+	return enterChroot(enter{
+		RepoRoot: RepoRoot(),
+		Chroot:   filepath.Join(RepoRoot(), chroot),
+		CmdDir:   chrootRepoRoot,
```

Comment:
`enterChroot` should default these fields if not set (or maybe override them entirely if they don't need to be customizable?), rather than open-coding their values everywhere.